Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM sonarsource/sonar-scanner-cli:11.1
+ARG CONTAINER_REGISTRY
+FROM ${CONTAINER_REGISTRY}/sonarsource/sonar-scanner-cli:11.1
```

Comment:
Maybe letting the leading `/` in the `CONTAINER_REGISTRY` arg and adding a default value as ""

Reply:
New scans seem to not pass using this rewrite, could someone please advise? Dockerhub throttling caused by this action is blocking all our CICD processes.

![image](https://github.com/user-attachments/assets/6f489ad7-8fc4-47de-b010-5fd2f5cfedf4)